src/__tests__/Select.test.tsx

```diff
@@ -0,0 +1,136 @@
+          <Select.Option value="four">Choice four</Select.Option>
+          <Select.Option value="five">Choice five</Select.Option>
+          <Select.Option value="six">Choice six</Select.Option>
+        </Select>
+      </>
+    )
+
+    const placeholderOption = getByText('Pick a choice')
+    const select = getByLabelText('Choice')
+
+    expect(select.getAttribute('aria-required')).toBeFalsy()
+
+    expect(placeholderOption).toBeDefined()
+    expect(placeholderOption.tagName.toLowerCase()).toBe('option')
+    // @ts-expect-error Property 'selected' does not exist on type 'HTMLElement'
+    expect(placeholderOption.selected).not.toBeNull()
+    expect(placeholderOption.getAttribute('disabled')).toBeNull()
+    expect(placeholderOption.getAttribute('hidden')).toBeNull()
+  })
+
+  it('renders a required select input with a placeholder', () => {
+    const {getByText, getByLabelText} = render(
+      <>
+        {/* eslint-disable-next-line jsx-a11y/label-has-for */}
+        <label htmlFor="reqWithPlaceholder">Choice</label>
+        <Select id="reqWithPlaceholder" placeholder="Pick a choice" required>
+          <Select.Option value="one">Choice one</Select.Option>
+          <Select.Option value="two">Choice two</Select.Option>
+          <Select.Option value="three">Choice three</Select.Option>
+          <Select.Option value="four">Choice four</Select.Option>
+          <Select.Option value="five">Choice five</Select.Option>
+          <Select.Option value="six">Choice six</Select.Option>
+        </Select>
+      </>
+    )
+
+    const placeholderOption = getByText('Pick a choice')
+    const select = getByLabelText('Choice')
+
+    expect(select.getAttribute('aria-required')).toBeTruthy()
+
+    expect(placeholderOption).toBeDefined()
+    expect(placeholderOption.tagName.toLowerCase()).toBe('option')
+    // @ts-expect-error Property 'selected' does not exist on type 'HTMLElement'
+    expect(placeholderOption.selected).not.toBeNull()
+    expect(placeholderOption.getAttribute('disabled')).not.toBeNull()
+    expect(placeholderOption.getAttribute('hidden')).not.toBeNull()
+  })
+
+  it('renders a disabled select input', () => {
+    const {getByLabelText} = render(
+      <>
+        {/* eslint-disable-next-line jsx-a11y/label-has-for */}
+        <label htmlFor="disabled">Choice</label>
+        <Select id="disabled" disabled>
+          <Select.Option value="one">Choice one</Select.Option>
+          <Select.Option value="two">Choice two</Select.Option>
+          <Select.Option value="three">Choice three</Select.Option>
+          <Select.Option value="four">Choice four</Select.Option>
+          <Select.Option value="five">Choice five</Select.Option>
+          <Select.Option value="six">Choice six</Select.Option>
+        </Select>
+      </>
+    )
+
+    const select = getByLabelText('Choice')
+
+    expect(select.getAttribute('aria-disabled')).toBeTruthy()
```

Comment:
❤️ . Personal preference, but`@testing-library/jest-dom` has some nice, more easily-readable matchers for this type of test. E.g. 

```suggestion
    expect(select).toHaveAttribute('aria-disabled')
```